### DISCUSSION TITLE
How to add compliance for multiple clients in one click

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [madrecha/jamku#622] [QA]
- How to add compliance to multiple clients in bulk?
- How to add another compliance to same set of Clients for which there exists a compliance?

**Comment by Divwork92** [comment]
A. Go to client list page
    select all the clients or set of clients by clicking on the square box which is extreme right side 
    click on add compliance 
    select service name from the drop down list, set periodicity, start date and end date
    click on save.

Second way is by importing
